src/main/java/pl/ug/virtualofficebackend/domain/user/entity/User.java

```diff
@@ -1,12 +1,72 @@
 package pl.ug.virtualofficebackend.domain.user.entity;
 
-import javax.persistence.Entity;
-import javax.persistence.Id;
+import pl.ug.virtualofficebackend.domain.entity.Workstation;
+
+import javax.persistence.*;
 import java.util.UUID;
 
-@Entity
+@Entity(name = "office_user")
 public class User {
-
     @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
     private UUID id;
```

Comment:
W jednych tabelach stosujemy UUID, a w innych longa.
Nie powinnno być wsżędzie long?

Reply:
i dont think so. UUID na user to imho jakiś sposób security. Na przedmiotach to jest useless.
Można to zmienić na Long ale to bez różnicy jaki rodzaj id bedziemy stosować.

Reply:
Wolałbym jednak ten long, pozwoli to na zachowanie jakiejś spójności,  a nie wiem, w jaki sposób to by miało być bezpieczniejsze.